### PR TITLE
PLANET-2857 User Profile: HTML not appearing correctly on mobile devices (M,S)

### PR DESCRIPTION
**Pull request by Wakkatela**
- Added specificity to remove floats from share buttons.
- Removed unnecessary markup.
- Removed duplicated user info block.

Still need to check that fits design. Is there a guide where I can get designs for layouts?

In this case, in desktop the share buttons are top right, alone and away from the main content while description is below the picture.  I moved the buttons top of description under author photo, but need to be sure is that way and not the description that needs to be top right too.

### CODE REVIEW

**Comment by comzeradd** [comment]
Are these changes related to [2857](https://jira.greenpeace.org/browse/PLANET-2857)?

As far as I understand, the bug report is about the Author bio not rendered properly on mobile when it has html tags (check the [screenshot](https://drive.google.com/file/d/1SxFPL2ng7YoK6rla-lcrBu_c3y12n2Q9/view)).

**Comment by comzeradd** [comment]
Ah right! Now, I get it :+1:

File: assets/scss/pages/_author.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
         }
       }
     }
+
+    .share-buttons {
+      float: none;
```
Comment (comzeradd) [inline assets/scss/pages/_author.scss]: That affects L, XL screens as well. We need the share buttons to float right there. See `_share-buttons.scss`.

**Comment by kirdia** [comment]
I removed them. Yes, they weren't really necessary .

**Pull request merged**